content/monitors/faq/as-count-monitor-evaluation.md

```diff
@@ -7,30 +7,30 @@ kind: faq
 
 Currently monitors that have the `as_count` modifier use a separate evaluation path than other monitors. In certain use-cases this this can result in unexpected behavior. We intend to migrate all monitors with `as_count` to the same evaluation path as other monitors, but this doc explains the underlying issue and will guide you through the migration process.
 
-Let's call the current evaluation path for `as_count` monitors `current_eva_path` and the new one `new_eva_path`.  
+Let's call the current evaluation path for `as_count` monitors `current_eval_path` and the new one `new_eval_path`.  
 
-The difference between those two evaluation paths is that complex monitor queries, especially those that have division or multiplication, may produce **unintended results** depending on the time aggregation function:
+The difference between those two evaluation paths is that complex monitor queries, especially those that have division, may produce **unintended results** depending on the time aggregation function. Let's say that you have 2 metrics, *A* and *B*:
```

Comment:
Again, carrying on Dmytro's comment, `pct_change` includes a division, could be worth mentioning ?